lightning/src/ln/channelmanager.rs

```diff
@@ -12044,6 +12194,137 @@ mod tests {
 			check_spends!(txn[0], funding_tx);
 		}
 	}
+
+	#[test]
+	fn test_peel_payment_onion() {
```

Comment:
Not to harp on this but good amount of trailing whitespace in these tests/utils

Reply:
ok, pretty i got em all